Cargo.toml

```
@@ -17,6 +17,12 @@
 [workspace]
 members = [".", "./macros", "./agones"]
 
+[workspace.dependencies]
+kube = { version = "0.77.0", features = ["derive", "runtime", "rustls-tls", "client"], default-features = false }
+k8s-openapi = { version = "0.16.0", features = ["v1_22", "schemars"] }
+tokio = { version = "1.23.0", features = ["rt-multi-thread", "fs", "signal", "test-util", "parking_lot", "tracing"] }
```

```suggestion
tokio = { version = "1.24.0", features = ["rt-multi-thread", "fs", "signal", "test-util", "parking_lot", "tracing"] }
```

_Just_ came out.

> The highlight of this release is the reduction of lock contention for all I/O operations ([#530](https://github.com/tokio-rs/tokio/pull/530)). We have received reports of up to a 20% improvement in CPU utilization and increased throughput for real-world I/O heavy applications.

😄  seems like that might be us.

---

Now it's 1.24.1 😄